src/script/view_model/content/ConnectRequestsViewModel.ts

```diff
@@ -17,32 +17,44 @@
  *
  */
 
-import {getLogger} from 'Util/Logger';
+import {getLogger, Logger} from 'Util/Logger';
 import {scrollToBottom} from 'Util/scroll-helpers';
 import {isLastItem} from 'Util/ArrayUtil';
 
 import {ParticipantAvatar} from 'Components/participantAvatar';
+import {MainViewModel} from '../MainViewModel';
+import {ContentViewModel} from '../ContentViewModel';
+import {UserRepository} from '../../user/UserRepository';
+import {ActionsViewModel} from '../ActionsViewModel';
+import {User} from '../../entity/User';
 
-window.z = window.z || {};
-window.z.viewModel = z.viewModel || {};
-window.z.viewModel.content = z.viewModel.content || {};
+type Repositories = {
+  user: UserRepository;
+};
 
-z.viewModel.content.ConnectRequestsViewModel = class ConnectRequestsViewModel {
+export class ConnectRequestsViewModel {
+  mainViewModel: MainViewModel;
+  userRepository: UserRepository;
+  logger: Logger;
+  actionsViewModel: ActionsViewModel;
+  connectRequests: ko.Computed<User[]>;
+  ParticipantAvatar: typeof ParticipantAvatar;
+  shouldUpdateScrollbar: ko.Computed<User[]>;
   /**
    * View model for connection requests.
    *
    * @param {MainViewModel} mainViewModel Main view model
    * @param {ContentViewModel} contentViewModel Content view model
    * @param {Object} repositories Object containing all repositories
    */
-  constructor(mainViewModel, contentViewModel, repositories) {
+  constructor(mainViewModel: MainViewModel, contentViewModel: ContentViewModel, repositories: Repositories) {
```

Comment:
You could define them right here as `private readonly mainViewModel: MainViewModel` etc. That would save you the need to declare them (line 36) and also save the this.mainViewModel = mainViewModel.